UnitWorld_shared/shared/game/GameManager.h

```diff
@@ -20,43 +22,49 @@ namespace uw
             _nextAddPlayer(nullptr)
```

Comment:
in `1000 / PHISICS_FRAME_PER_SECOND`, `1000 / ...` should be part of the `PHISICS_FRAME_PER_SECOND` constant

Reply:
It is constant, but I think this is pretty logic by itself. Sure I'd use a helper `framePerSecondToMsPerFrame` if I had a timing helper class. Should I do that?